help/maintainer/docs-release-process.md

```diff
@@ -5,136 +5,41 @@ weight: 75
 
-  This is really just for builds you do on your machine, but update it on github as well to keep everything in sync. Set line 20
-   to the version you just created.
-![local](https://user-images.githubusercontent.com/35748459/87464508-13878100-c5c8-11ea-840f-25e4ab80e372.png)
-* Configure staging builds: [/website/config/staging/params.toml](https://github.com/knative/website/blob/main/config/staging/params.toml)
 
-  Set line 19 to the version you just created.
-![staging](https://user-images.githubusercontent.com/35748459/87464866-afb18800-c5c8-11ea-9ce0-74331523d651.png)
-* Define the default branch variable:[/website/scripts/docs-version-settings.sh](https://github.com/knative/website/blob/main/scripts/processsourcefiles.sh)
+## Update the main branch to reference the release
 
-  This variable should be set to the version you just created.
-![doc-version](https://user-images.githubusercontent.com/35748459/87465326-4bdb8f00-c5c9-11ea-95c7-8a9e3b8abecd.png)
-* Tell Netlify how to get the previous three releases by updating the three git clone commands to point to those three releases: [/website/scripts/processsourcefiles.sh](https://github.com/knative/website/blob/main/scripts/processsourcefiles.sh)
-![processsource](https://user-images.githubusercontent.com/35748459/87465528-ad9bf900-c5c9-11ea-8364-d391c1926332.png)
+### `hack/build.sh`
 
-* Add the just created version to line 24 of the Netlify toml file.  The :splat keeps the links to the three previous versions
-  alive, so the current version + the previous 3 versions that are still being built. The bottom release that fell out of the top
-  four should be added to the redirects below line 29.  Any links to these versions are redirected back to the most current
-  version: [/website/netlify.toml](https://github.com/knative/website/blob/main/netlify.toml)
-![netlify](https://user-images.githubusercontent.com/35748459/87465963-54809500-c5ca-11ea-8372-3fbcfc965e20.png)
+Find the references to the previous release number and update the lines to
+include the new release number as well. We keep the last 4 releases available
+per [our support
+window](https://github.com/knative/community/blob/main/mechanics/RELEASE-VERSIONING-PRINCIPLES.md#knative-community-support-window-principle),
+so clean up old versions at the end of the list when there will be more than 4.
 
-## Test the content
-* Update your local versions of the build config files to run a local build.
-* Run a local build using from the /website directory: scripts/localbuild.sh. Follow the extensive instructions at the top of the script file.
-* Check to make sure everything looks okay.
-* Sync your changes to the staging branch with git pull --rebase upstream main and git push upstream staging. {need more detail}
+Until the old hugo configuration has been aged out (after release 0.27), it's
+best to send these PRs to @julz to double-check. You'll also want to remove the
```

Comment:
happily there's only one more release before this happens... and I'm one of the release leads for it, so this clause may 🤞 never have to be used, other than via internal monologue.